cmd/podman/containers/cp.go

```diff
@@ -345,10 +345,15 @@ func copyToContainer(container string, containerPath string, hostPath string) er
 		isStdin = true
 	}
 
-	// Make sure that host path exists.
-	hostInfo, err := copy.ResolveHostPath(hostPath)
-	if err != nil {
-		return fmt.Errorf("%q could not be found on the host: %w", hostPath, err)
+	hostInfo := &copy.FileInfo{}
+	var err error
+
+	if !isStdin {
```

Comment:
this could just be an else on the conditional above instead of a new if

Reply:
Simplified into a single if/else with the stdin check.